binding-service-impl/src/main/java/io/mosip/esignet/services/KeyBindingServiceImpl.java

```diff
@@ -69,18 +74,29 @@ public BindingOtpResponse sendBindingOtp(BindingOtpRequest bindingOtpRequest, Ma
 			log.error("Failed to send binding otp: {}", e);
 			throw new EsignetException(e.getErrorCode());
 		}
-
 		if (sendOtpResult == null) {
 			log.error("send-otp Failed wrapper returned null result!");
 			throw new EsignetException(SEND_OTP_FAILED);
 		}
-
 		BindingOtpResponse otpResponse = new BindingOtpResponse();
 		otpResponse.setMaskedEmail(sendOtpResult.getMaskedEmail());
 		otpResponse.setMaskedMobile(sendOtpResult.getMaskedMobile());
 		return otpResponse;
 	}
 
+	@Override
+	public BindingOtpResponse sendBindingOtpV2(BindingOtpRequestV2 bindingOtpRequestV2, Map<String, String> requestHeaders)
+			throws EsignetException {
+		captchaHelper.validateCaptchaToken(bindingOtpRequestV2.getCaptchaToken(), "binding-otp");
+
+		BindingOtpRequest bindingOtpRequest = new BindingOtpRequest();
+		bindingOtpRequest.setIndividualId(bindingOtpRequestV2.getIndividualId());
+		bindingOtpRequest.setOtpChannels(bindingOtpRequestV2.getOtpChannels());
```

Comment:
There is no need to construct a new instance, as BindingOtpRequest is the parent class of BindingOtpRequestV2 @Md-Humair-KK